### PR TITLE
[email] Show attachments in email condensed and expanded views

### PR DESCRIPTION
**Pull request by heathergh**
# Code changes

- Added conditional logic to `initializeAttachedFiles()` in the `Email` and `MessageBody` components to filter out attachments with a `content_id` that matches an element in the `cids` array

# Screenshots
## Before
Screen Shot 2022-01-25 at 4 07 50 PM
Screen Shot 2022-01-25 at 4 11 40 PM

## After
Screen Shot 2022-01-25 at 4 07 09 PM
Screen Shot 2022-01-25 at 4 11 21 PM

# Readiness checklist
- [ ] Added changes to component `CHANGELOG.md`
- [ ] New property added? make sure to update `component/src/properties.json`
- [ ] Cypress tests passing?
- [ ] New cypress tests added?
- [ ] Included before/after screenshots, if the change is visual

# License

I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: components/email/src/Email.svelte [hunk 1]
```diff
@@ -1880,7 +1885,8 @@
                   {/if}
                   <span
                     class="snippet"
-                    class:deleted={activeThread.messages.length <= 0 && !isDraft}
+                    class:deleted={activeThread.messages.length <= 0 &&
+                      !isDraft}
```
Comment (heathergh) [inline components/email/src/Email.svelte:1889]: whitespace change

File: commons/src/components/MessageBody.svelte [hunk 2]
```diff
@@ -33,11 +33,14 @@
       for (const [fileIndex, file] of message.files.entries()) {
         if (
           file.content_disposition === "attachment" &&
-          !(file.content_id && InlineImageTypes.includes(file.content_type)) && // treat all files with content_id as inline
+          !(
+            file.content_id &&
+            message.cids?.includes(file.content_id) &&
+            InlineImageTypes.includes(file.content_type)
+          ) &&
```
Comment (pooja169usp) [inline commons/src/components/MessageBody.svelte:40]: This condition check can be extracted into a function for better readability.

**Review by pooja169usp — APPROVED** [review]
Looks good 👍🏽

File: commons/src/components/MessageBody.svelte (hunk 2 above)
Comment (heathergh) [reply commons/src/components/MessageBody.svelte:40]: Good point! I will do that :)

Comment (heathergh) [reply commons/src/components/MessageBody.svelte]: Done https://github.com/nylas/components/pull/359/commits/82a52d9108ba3e53ba07e49e470c2838c73fbf35

File: commons/src/methods/isNonInlineAttachment.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import type { Message, File } from "@commons/types/Nylas";
+import {
+  DisallowedContentTypes,
+  InlineImageTypes,
+} from "@commons/constants/attachment-content-types";
+
+export const isNonInlineAttachment = (message: Message, file: File): boolean =>
```
Comment (AaronDDM) [inline commons/src/methods/isNonInlineAttachment.ts:7]: Only thing is this function name - `isNon` is strange. But I can't seem to think of a better name :( 

Would be ideal if we could detect `isInlineAttachment` and then just use the negative.

**Review by AaronDDM — APPROVED** [review]

File: commons/src/methods/isNonInlineAttachment.ts (hunk 3 above)
Comment (heathergh) [reply commons/src/methods/isNonInlineAttachment.ts:7]: @AaronDDM I think this can be more succinctly named `isAttachedFile`. WDYT?

Comment (AaronDDM) [reply commons/src/methods/isNonInlineAttachment.ts:7]: Yes!

**Pull request merged**